src/lib_ccx/params.c

```diff
@@ -1198,6 +1198,11 @@ int parse_parameters (struct ccx_s_options *opt, int argc, char *argv[])
 			opt->hardsubx = 1;
 			continue;
 		}
+		if (strcmp(argv[i], "-letterwise_russian")==0)
+		{
+			opt->letter_russian = 1;
+			continue;
```

Comment:
Put all the things related to Russian OCR together in params.c with clear commenting about what each is. Also add the usage instructions to the help screen text.

Reply:
Noted